spec/duration.html

```diff
@@ -782,7 +782,7 @@ <h1>Duration Records</h1>
       <h1>Date Duration Records</h1>
       <p>
         A <dfn variants="Date Duration Records">Date Duration Record</dfn> is a Record value used to represent the portion of a Temporal.Duration object that deals with calendar date units.
-        Date Duration Records are produced by the abstract operations BalanceDurationRelative and UnbalanceDurationRelative.
+        Date Duration Records are produced by the abstract operation CreateDateDurationRecord, among others.
```

Comment:
"abstract operation Foo" or "Foo abstract operation"? (Doesn't matter much, but I noticed you were inconsistent.)